components/ws-proxy/pkg/proxy/routes.go

```diff
@@ -79,9 +79,10 @@ func installWorkspaceRoutes(r *mux.Router, config *RouteHandlerConfig, ip Worksp
 	// The favicon warants special handling, because we pull that from the supervisor frontend
 	// rather than the IDE.
 	faviconRouter := r.Path("/favicon.ico").Subrouter()
+	faviconPath := "/_supervisor/frontend/favicon.ico"
 	faviconRouter.Use(func(h http.Handler) http.Handler {
 		return http.HandlerFunc(func(resp http.ResponseWriter, req *http.Request) {
-			req.URL.Path = "/_supervisor/frontend/favicon.ico"
+			req.URL.Path = faviconPath
```

Comment:
Do not override existing icon files.

Reply:
Here its not overriding the file. I am just mainting a path variable for the same . User can just switch the path as desired

Reply:
As less code changes as possible to main repo. We are not in code cleanup mode or future flexibility mode.